packages/components/tests/integration/components/hds/breadcrumb/index-test.js

```diff
@@ -22,6 +22,15 @@ module('Integration | Component | hds/breadcrumb/index', function (hooks) {
     assert.dom('#test-breadcrumb').hasClass('hds-breadcrumb--items-can-wrap');
   });
 
+  test('it should dispatch a didInsert event when the component is rendered', async function (assert) {
+    let inserted = false;
+    this.set('didInsert', () => (inserted = 'true'));
```

Comment:
```suggestion
    this.set('didInsert', () => (inserted = true));
```

Reply:
good call. made this update and pushed up the branch again.